release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2720,6 +2720,78 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[source,terminal]
+----
+$ oc adm release info 4.15.12 --pullspecs
+----
+
+[id="ocp-4-15-12-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-15-12-apiversion-upgrade-v1beta1"]
+===== API Version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener upgraded from v1alpha1 to v1beta1
+
+* Previously, the API version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener was v1alpha1. With this release, the API version is upgraded to v1beta1 so that the pipelines plugin supports the latest Pipeline Trigger API version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener. (link:https://issues.redhat.com/browse/OCPBUGS-31445[*OCPBUGS-31445*])
+
+[id="ocp-4-15-12-pipeline-list-page-improvement"]
+===== PipelineRun list view performance improvement
+
+* Previously, in the PipelineRun list page, all of the TaskRuns were fetched and separated based on their PipelineRun name. With this release, TaskRuns are only fetched for failed and cancelled PipelineRuns, and a caching mechanism is added to fetch the PipelineRuns and TaskRuns that are associated with the failed and cancelled PipelineRuns. (link:https://issues.redhat.com/browse/OCPBUGS-31799[*OCPBUGS-31799*])
+
+[id="ocp-4-15-12-installer-handling-new-character"]
+===== Installer handles the escaping of the "%" character
+
+* Previously, if a cluster was installed using proxy and the proxy information contained escaped characters in the format "%XX", the installation failed. With this release, the Installer now handles the escaping of the "%"" character. (link:https://issues.redhat.com/browse/OCPBUGS-32259[*OCPBUGS-32259*])
+
+[id="ocp-4-15-12-cfe-evaluation-status"]
+===== Cluster Fleet Evaluation status information added to the Machine Config Operator
+
+* Previously, the Machine Config Operator (MCO) did not include the Cluster Fleet Evaluation (CFE) Status. With this release, the CFE status information is added to the MCO and available to customers. (link:https://issues.redhat.com/browse/OCPBUGS-32922[*OCPBUGS-32922*])
+
+[id="ocp-4-15-12-operatorhub-filter-rename-FIPS"]
+===== OperatorHub filter renamed from "FIPS Mode" to "Designed for FIPS"
+
+* Previously, OperatorHub included a filter named "FIPS Mode". With this release, that filter is named "Designed for FIPS". (link:https://issues.redhat.com/browse/OCPBUGS-32933[*OCPBUGS-32933*])
+
+[id="ocp-4-15-12-bug-fixes"]
+==== Bug fixes
+
+* Previously, containers had an incorrect view of the pids limit in their `cgroup` hierarchy and reported as a random number instead of `max`. The containers do not have max pids and are limited by the pod pid limit, which is set outside of the container's `cgroup` hierarchy and not visible from within the container. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-28926[*OCPBUGS-28926*])
```

Comment:
```suggestion
* Previously, containers had an incorrect view of the pids limit in their `cgroup` hierarchy and reported as a random number instead of `max`. The containers do not have max PIDs and are limited by the pod PID limit, which is set outside of the container's `cgroup` hierarchy and not visible from within the container. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-28926[*OCPBUGS-28926*])
```